src/status_im/subs.cljs

```diff
@@ -1838,40 +1846,58 @@
    (:show-name? multiaccount)))
 
 (re-frame/reg-sub
- :ens.registration/screen
+ :ens/search-screen
+ :<- [:ens/registration]
+ (fn [{:keys [custom-domain? username state]}]
+   {:state          state
+    :username       username
+    :custom-domain? custom-domain?}))
+
+(defn- ens-amount-label
```

Comment:
Can we move this fn to another namespace?

Reply:
this is a private function that is only useful here